pkg/console/controllers/route/controller.go

```diff
@@ -128,12 +128,14 @@ func (c *RouteSyncController) Sync(ctx context.Context, controllerContext factor
 	// route into inaccessible state.
 	_, customRouteErrReason, customRouteErr := c.SyncCustomRoute(ctx, routeConfig, controllerContext)
 	statusHandler.AddConditions(status.HandleProgressingOrDegraded("CustomRouteSync", customRouteErrReason, customRouteErr))
+	statusHandler.AddCondition(status.HandleUpgradable("DefaultRouteSync", customRouteErrReason, customRouteErr))
```

Comment:
Isn't this once again `CustomRouteSync`?

Do you want to distinguish the error types for upgradable and degraded cases?

Reply:
We are distinguishing between them, since we are calling both:
- `HandleProgressingOrDegraded`, which will set operator's `Degraded` condition to `True` if an error occurs
- `HandleUpgradable`, which will set operator's `Upgradable` condition to `False` if an error occurs